sbin/umount/umount.c

```diff
@@ -139,6 +153,8 @@ main(int argc, char *argv[])
 			errx(1, "%s: %s", nfshost, gai_strerror(error));
 	}
 
+	multiple = all;
```

Comment:
why do we need a new name for this?

Reply:
> why do we need a new name for this?

I did it mainly to avoid making `all` a global variable, but I agree that it's a much better option.